src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -219,28 +214,6 @@ class FilterBar extends Component {
         label: getLabel(l10n.getStr("webconsole.debugFilterButton.label"), FILTERS.DEBUG),
         filterKey: FILTERS.DEBUG,
         dispatch,
-      }),
-      dom.div({
-        className: "devtools-separator",
-      }),
-      FilterButton({
```

Comment:
Should we update the `FILTERS` properties in `/src/devtools/client/webconsole/constants.js` as well?

Reply:
we should write a script that finds unused properties!